src/layouts/index.css

```diff
@@ -205,6 +205,12 @@ footer a {
   ##Screen = B/w 320px to 479px
 */
 
+@media (max-width: 395px) {
```

Comment:
It looks like the issue still exists on screen widths between 390px to 400px. It'd be good to set the break point to 400px? do check it out :smile: